.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+---
+
+## Checklist:
+
+- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
+  minor change.
+  - [ ] No
+    - [ ] The change in this pull request is minor.
+  - [ ] Yes
+    - Close #{issue_number}
+
+For the following questions, only check the boxes that are applicable.
+
+- [ ] Add descriptive pull request text.
+- [ ] Add/update relevant tests.
+- [ ] Add/update relevant documents.
+
+- Do you have triage permission of this repository?
+  - [ ] No
+    - You are good.
+  - [ ] Yes
+    - Does this pull request close an issue?
+      - [ ] No, this pull request also acts as an issue by itself.
+        - [ ] Assign yourself.
+        - [ ] Label this pull request.
+        - [ ] Put a milestone.
+        - [ ] Put this pull request under "In Progress" or "Under Review" pipeline on ZenHub.
+        - [ ] Put this pull request under the appropriate epic on ZenHub.
```

Comment:
How do you add a PR under a Zenhub epic? (maybe we should add a brief explanation here for those unfamiliar on how to do it too)

Reply:
It's the same as issues, what text would you suggest?

Reply:
Oh ok never mind, I know what happened now. I removed the Zenhub extension because it doesn't mesh well with GitHub dark mode, but as a result I don't see the Epic or Estimate options on PR's. My bad.

Reply:
You don't have to do it on this page, the checklist is meant to be ticked after the PR is opened, so you would have an opportunity to head to the ZenHub user interface. You can't tick the boxes before the PR is opened anyway.